### DISCUSSION TITLE
sensor add error

### DISCUSSION THREAD

**Discussion by 73F0SG1R** [Security-Onion-Solutions/securityonion#3236] [Unsupported Versions]
ISO 2.3.21

![image](https://user-images.githubusercontent.com/31665237/109694031-8f2e4200-7b47-11eb-97e6-10a81591ef31.png)

SO manager username/password are correct:

difference from when successfully added sensor(s) yesterday:     sudo soup (bunch of updates yesterday)

any help would be appreciated

thanks

**Comment by cm-ops** [comment]
Is this a reinstall? Try removing `/root/manager_setup` and running so-setup again.
